src/hooks/useTransactions/useTransactions.ts

```diff
@@ -9,12 +8,12 @@ import { useStateIfMounted } from 'hooks/useStateIfMounted/useStateIfMounted'
 import { fromBaseUnit } from 'lib/math'
 dayjs.extend(relativeTime)
 
-export type FormatTransactionType = Transaction & {
+export type FormatTransactionType = Transaction<ChainTypes> & {
```

Comment:
No need to specify `ChainTypes` with that being the default for generic type
```suggestion
export type FormatTransactionType = Transaction & {
```